src/article/article.ts

```diff
@@ -103,21 +103,45 @@ const getHeader = (articleDom: DocumentFragment): string => {
   replaceAttributesWithClassName(articleDom, 'article > [data-itemprop="affiliations"]', 'content-header__affiliations');
   replaceAttributesWithClassName(articleDom, '.content-header__affiliations > [itemtype="http://schema.org/Organization"]', 'organisation');
   replaceAttributesWithClassName(articleDom, '.organisation > address', 'organisation__address');
-  replaceAttributesWithClassName(articleDom, 'article > [data-itemprop="identifiers"]', 'content-header__identifiers');
 
   const headline = articleDom.querySelector('.content-header__title');
   const authors = articleDom.querySelector('.content-header__authors');
   const affiliations = articleDom.querySelector('.content-header__affiliations');
-  const identifiers = articleDom.querySelector('.content-header__identifiers');
+  const dois = getDois(articleDom);
 
   return `<div class="content-header">
     ${headline?.outerHTML}
     ${authors?.outerHTML}
     ${affiliations?.outerHTML}
-    ${identifiers?.outerHTML}
+    <div class="content-header__footer">
+      ${dois}
```

Comment:
There would only one doi per page so should we rename this to doi?

Reply:
Ah, sure. I didn't know to expect that, and Chris's design has "show all" in it. I've changed to expect (and test) for a single DOI, and make sure we return just one, but still as a list for now, until I figure out what show all is for.